oteljava/metrics/src/main/scala/org/typelevel/otel4s/oteljava/metrics/HistogramBuilderImpl.scala

```diff
@@ -112,15 +112,17 @@ object HistogramBuilderImpl {
 
             def recordDuration(
                 timeUnit: TimeUnit,
-                attributes: immutable.Iterable[Attribute[_]]
+                attributes: Resource.ExitCase => immutable.Iterable[
+                  Attribute[_]
+                ]
```

Comment:
I hate so much that this is how it formats

Reply:
Same. A lot of places have this awful formatting :( 

Perhaps we should increase `maxColumn` to 100.

Reply:
I am 100% in favour

Reply:
Among the typelevel projects for **maxColumn** the following values are most often used:
* 27 project use **100** https://github.com/search?q=org%3Atypelevel++path%3A.scalafmt.conf+%22maxColumn+%3D+100%22&type=code
*  21 projects use **120** for https://github.com/search?q=org%3Atypelevel++path%3A.scalafmt.conf+%22maxColumn+%3D+120%22&type=code

Reply:
@mzuehlke what about the number using the default?

Reply:
Here is what I found:
- 79 projects have `.scalafmt.conf` file https://github.com/search?q=org%3Atypelevel++path%3A.scalafmt.conf+&type=code
- 59 have a custom `maxColumn` https://github.com/search?q=org%3Atypelevel++path%3A.scalafmt.conf+%22maxColumn%22&type=code
- 1 have `maxColumn = 80` https://github.com/search?q=org%3Atypelevel++path%3A.scalafmt.conf+%22maxColumn+%3D+80%22&type=code

So, 21 projects use the default `80`.